packages/zambdas/src/get-slug-availability/index.ts

```diff
@@ -9,11 +9,12 @@ export const index = async (input: ZambdaInput): Promise<APIGatewayProxyResult>
 };
 
 interface getSlugAvailabilityInput {
+  practitionerId?: string;
   slug: string;
 }
 
 const getSlugAvailability = async (input: ZambdaFunctionInput): Promise<ZambdaFunctionResponse> => {
-  const { slug } = input.body as getSlugAvailabilityInput;
+  const { slug, practitionerId } = input.body as getSlugAvailabilityInput;
```

Comment:
Updating the backend in this way feels overly complex when it's really more of a FE problem to solve. For example, we could check if the slug the user entered is the same as their current slug - in that case we don't send an API request and indicate to the user that this is their current slug.

Reply:
makes sense will update logic